.github/workflows/checks.yml

```
@@ -110,11 +110,11 @@ jobs:
         run: npm run lint
   fuzz:
     name: Fuzz
-    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@main
+    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@job-based-timeouts-for-fuzzing
```

This should be reverted prior to merging:

```suggestion
    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@main
```